.github/workflows/publish-technical-documentation-release.yml

```diff
@@ -0,0 +1,29 @@
+name: publish-technical-documentation-release
+
+on:
+  push:
+    branches:
+      - 'release/*'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+    paths:
+      - 'docs/sources/**'
+  workflow_dispatch:
+jobs:
+  sync:
+    if: github.repository == 'grafana/alloy'
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: grafana/writers-toolkit/publish-technical-documentation-release@publish-technical-documentation-release/v1
+        with:
+          release_tag_regexp: "^v(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)$"
+          release_branch_regexp: "^release/v(0|[1-9]\\d*)\\.(0|[1-9]\\d*)$"
+          release_branch_with_patch_regexp: "^release/v(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)$"
+          website_directory: content/docs/alloy
+          version_suffix: ""
```

Comment:
I suppose this will update both `content/docs/alloy/latest` and, for example, `content/docs/alloy/v1.2/` ?

Reply:
Sorry, late reply. 

Short answer: yes.

Longer answer:

We actually publish only version directories and then the website derives "latest" from the highest published version directory. So `content/docs/alloy/latest` doesn't actually get updated but `/docs/alloy/latest/` does.